src/pages/chat-page/ChatPage.tsx

```diff
@@ -1,22 +1,32 @@
 import ChatContent from '@metis/features/channels/components/chat-content/ChatContent';
 import ChatFooter from '@metis/features/channels/components/chat-footer/ChatFooter';
 import ChatHeader from '@metis/features/channels/components/chat-header/ChatHeader';
+import EmptyStateChannel from '@metis/features/channels/components/empty-state-channel/EmptyStateChannel';
 import FooterSection from '@metis/layout/sections/FooterSection';
 import HeaderSection from '@metis/layout/sections/HeaderSection';
 import MainSection from '@metis/layout/sections/MainSection';
+import { useAppSelector } from '@metis/store/hooks';
 
-const ChatPage = () => (
-  <>
-    <HeaderSection>
-      <ChatHeader />
-    </HeaderSection>
-    <MainSection>
-      <ChatContent />
-    </MainSection>
-    <FooterSection>
-      <ChatFooter />
-    </FooterSection>
-  </>
-);
+const ChatPage = () => {
```

Comment:
I suggest to use the router here. You can create a new route for empty state, and upon channel click you can navigate to the chat route passing an id, which will load your channel content.

Reply:
Done